tests/test_xet_download.py

```diff
@@ -293,3 +294,42 @@ def test_snapshot_download_cache_reuse(self, tmp_path):
 
             # Verify xet_get was not called (files were cached)
             mock_xet_get.assert_not_called()
+
+    def test_download_backward_compatibitily(self, tmp_path):
+        """Test that xet download works with the old pointer file protocol.
+
+        Until the next major version of hf-xet is released, we need to support the old
```

Comment:
When do you plan to make next major release of hf_xet? (to get an idea of how long this test should exist)

Reply:
v2 is not on our roadmap at the moment, so it's hard to say conclusively. I would guesstimate that we'll need to make a breaking change within the next 4 months which will force a v2 even if we don't have a v2 feature set by then.

Once that happens, we'll need to update `setup.py`. This test will fail during that CI run and will need to be removed. In other words, it won't outlive the `hf_xet` major version update.